### ISSUE TITLE
Running given example showing blank screen on android

### ISSUE THREAD

**Issue by raginisahu** [Michaelvilleneuve/react-native-document-scanner#81]
i am running the example that given in readme can you please help me or give the example how can i use this npm

**Comment by priyankagupta23pg** [comment]
yes I am also seeing blank screen in my app using this example in android

**Comment by retyui** [comment]
Error:

ViewManagerPropertyUpdater: Could not find generated setter for class com.documentscanner.DocumentScannerViewManager

**Comment by projitroy** [comment]
set flex:1 for both  
`<View style={{flex: 1}}> `
and 
`<DocumentScanner `style={{flex:1}}/>``

**Comment by mohdaamir8182** [comment]
UPDATE ON ISSUE:

>: apply @projitroy  above solution . This will solve the white screen issue.
>: If you see black screen then it is due to permissions. Allow the permissions manually or using a library totally up to you.
>: after allowing permissions everything will work fine.
